docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
```

Comment:
Having a docker-compose is nice. Can you add a section on how to run the connector using it in the README.